docs/create_equinix_cluster.md

```diff
@@ -0,0 +1,109 @@
+# Create Equinix Metal cluster using CAPI / CAPEM
+
+## Overview
+
+- We are using machines contributed to the CNCF by Equinix Metal.
+- These steps manually create a cluster using CAPI / CAPEM using a local
+Kind cluster for the management cluster.
+- These docs are based on the Equinix CAPI [guide](https://deploy.equinix.com/developers/guides/kubernetes-cluster-api/).
+- Using [Podman](https://github.com/kubernetes-sigs/cluster-api-provider-packet/issues/496)
+is recommended by Equinix for MacOS.
+- We plan to automate these steps later using GitHub Actions and an IaC tool.
+
+## Create Cluster
+
+- Log in to the Equinix Metal [console](https://console.equinix.com/) and get
+the project API key from the project settings. If it doesn't exist create it.
+
+```sh
+export PACKET_API_KEY="<PROJECT_API_KEY>"
+```
+
+- Install the CAPI controllers using [clusterctl](https://cluster-api.sigs.k8s.io/user/quick-start.html#install-clusterctl).
+
+```sh
+clusterctl init --infrastructure packet
+```
+
+- Set env vars with cluster config.
+
+```sh
+# Get the project ID from the project settings in the console
+export PROJECT_ID="<PROJECT_ID>"
+
+# Use Paris metro (Equinix region)
+export METRO="pa"
+
+# Use Ubuntu 22.04 with cGroup v2 for Kepler
+export NODE_OS="ubuntu_22_04"
+
+# The pod and service CIDRs for the new cluster
+export POD_CIDR="192.168.0.0/16"
+export SERVICE_CIDR="172.26.0.0/16"
+
+# Use node type with Intel CPUs for RAPL support
+export CONTROLPLANE_NODE_TYPE="m3.small.x86"
+export WORKER_NODE_TYPE="m3.small.x86"
```

Comment:
I think its safest to use a node type with an Intel CPU. I tested with an AMD CPU and Kepler couldn't access RAPL.

![Screenshot 2023-10-18 at 11 15 34](https://github.com/cncf-tags/green-reviews-tooling/assets/311527/b8b3cf24-5cee-4120-a3ec-9cea4dae4cf3)

I think this node type have good enough resources for now. WDYT?

See https://deploy.equinix.com/product/bare-metal/servers/

Reply:
+1 for using an Intel CPU and starting with `small`! 👍